backend/Dockerfile

```diff
@@ -1,18 +1,5 @@
-FROM python:3.11-slim
+FROM quay.io/jupyter/datascience-notebook:latest
```

Comment:
Would this break trulens. Ttuelens needs 3.11 to work. 3.12 I think is not yet compatible.